warehouse/scripts/run_and_upload.py

```diff
@@ -264,34 +264,34 @@ def get_command(*args) -> List[str]:
             results_to_check.append(subprocess.run(args))
 
     if sync_metabase:
-        for schema, database in [
-            ("views", "Data Marts (formerly Warehouse Views)"),
-            ("gtfs_schedule", "GTFS Schedule Feeds Latest"),
-            ("mart_transit_database", "Data Marts (formerly Warehouse Views)"),
-            ("mart_gtfs_guidelines", "Data Marts (formerly Warehouse Views)"),
-            ("mart_gtfs", "Data Marts (formerly Warehouse Views)"),
-            ("mart_gtfs_quality", "Data Marts (formerly Warehouse Views)"),
-            ("mart_audit", "Data Marts (formerly Warehouse Views)"),
-        ]:
-            args = [
-                "dbt-metabase",
-                "models",
-                "--dbt_manifest_path",
-                "./target/manifest.json",
-                "--dbt_database",
-                "cal-itp-data-infra",
-                "--dbt_schema",
-                schema,
-                "--metabase_host",
-                "dashboards.calitp.org",
-                "--metabase_user",
-                os.environ["METABASE_USER"],
-                "--metabase_password",
-                os.environ["METABASE_PASSWORD"],
-                "--metabase_database",
-                database,
-            ]
-            results_to_check.append(subprocess.run(args))
+        results_to_check.append(
+            subprocess.run(
+                [
+                    "dbt-metabase",
+                    "models",
+                    "--dbt_manifest_path",
+                    "./target/manifest.json",
+                    "--metabase_database",
+                    "Data Marts (formerly Warehouse Views)",
+                    "--dbt_schema_excludes",
```

Comment:
I am a little nervous about changing from an inclusion list to an exclusion list, doesn't this mean that any new schema will be included by default? Is that what we want?

Reply:
Hmm that's a good point; my assumption was that we'd really only have new mart schemas and we'd generally sync them to the data marts DB in Metabase. (Also it's only 1 CLI call instead of N). I was trying to avoid the silent failure case of someone creating a new schema, adding it in Metabase, and not seeing docs update. Do you have thoughts on that vs erroring when a new schema is added, and explicitly ignoring if it isn't set in Metabase?

Reply:
(we are discussing this offline)

Reply:
Ok, I misunderstood what this controls. This just syncs documentation & column metadata annotations. The core setting of what schemas' **data** to actually sync to Metabase is exclusively in the Metabase UI. So there isn't a risk here of **data** being exposed in Metabase unintentionally.